src/about/help.gemini

```diff
@@ -288,11 +288,16 @@ This list contains all built-in shortcuts:
 * Ctrl+D ⇒ Quick add/remove from favourites
 * Ctrl+L ⇒ Focus URL bar
 * Ctrl+S ⇒ Save current file
-* Ctrl+H ⇒ Go to home page
+* Ctrl+B ⇒ Toggle favourites dock
+* Ctrl+H ⇒ Toggle history dock
+* Ctrl+M ⇒ Toggle document outline dock
 * Ctrl+U ⇒ View document source
+* Ctrl+Q ⇒ Quit Kristall
+* Ctrl+, ⇒ Open settings window
```

Comment:
The Alt+, is a duplicate 
I like your wording better so you should remove the one on line 302 😉